sklearn/svm/tests/test_sparse.py

```diff
@@ -226,8 +232,14 @@ def test_error():
     assert_array_equal(clf.predict(T), true_result)
 
 
-def test_linearsvc():
+@pytest.mark.parametrize(
+    "lil_container, dok_container", zip(LIL_CONTAINERS, DOK_CONTAINERS)
```

Comment:
Is it possible to test all combinations using `itertools.product` instead?

Reply:
I think in most other tests, we are only doing `zip` but not `product`. Also in this case, I think the tests on `X_sp` and `X2_sp` are independent? So it may not be necessarily to test all combinations (if I did not overlook anything).